src/items/items.controller.ts

```diff
@@ -1,9 +1,9 @@
-import { Body, Controller, Delete, Get, Param, Post } from '@nestjs/common';
-import { CreateItemDto } from './dto/create-item.dto';
-import { Item } from './item.entity';
-import { ItemsService } from './items.service';
+import { Body, Controller, Delete, Get, Param, Post } from "@nestjs/common";
```

Comment:
These double quotes should just be left as single quotes. Similar comment for the other changed files here